src/app/config.rs

```diff
@@ -284,6 +292,12 @@ const ENV_DNS_MAX_TTL: &str = "LINKERD2_PROXY_DNS_MAX_TTL";
 /// an uncanonicalized address.
 const ENV_DNS_CANONICALIZE_TIMEOUT: &str = "LINKERD2_PROXY_DNS_CANONICALIZE_TIMEOUT";
 
+/// Configure the stream or connection level flow control setting for HTTP2.
+///
+/// If unspecified, the default value of 65,535 is used.
+const ENV_INITIAL_STREAM_WINDOW_SIZE: &str = "LINKERD2_PROXY_INITIAL_STREAM_WINDOW_SIZE";
+const ENV_INITIAL_CONNECTION_WINDOW_SIZE: &str = "LINKERD2_PROXY_INITIAL_CONNECTION_WINDOW_SIZE";
```

Comment:
As much as I don't like making the already long env var names even longer, I feel like these should be `LINKERD2_PROXY_HTTP2_INITIAL_STREAM_WINDOW_SIZE`/`LINKERD2_PROXY_HTTP2_INITIAL_CONNECTION_WINDOW_SIZE`, to differentiate them from the TCP flow control window?